.github/workflows/examples.yaml

```
@@ -9,8 +9,8 @@ jobs:
       - uses: actions/checkout@v3
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: '3.0.0'
-          channel: 'stable'
+          flutter-version: "3.0.0"
+          channel: "stable"
```

```suggestion
          flutter-version: '3.0.0'
          channel: 'stable'
```
I don't think we need to change this